.github/workflows/operator_ci.yaml

```diff
@@ -0,0 +1,184 @@
+name: Operator Build
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    branches:
+      - develop
+      - master
+jobs:
+  Prepare_env:
+   name: "Prepare env"
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - uses: FranzDiebold/github-env-vars-action@v2
+      - name: Setup dependencies
+        id: setup_dependencies
+        run: |
+          build/ci/github_actions/setup_ci_dependencies.sh
+        env:
+          build_number: "${{ github.run_number }}"
+      - uses: actions/cache@v2
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ env.pythonLocation }}-${{ hashFiles('dev-requirements.txt') }}
+      - name: Save dependencies files
+        uses: actions/upload-artifact@v2
+        with:
+          name: ci-dependencies
+          path: |
+            /home/runner/.bash_profile
+          retention-days: 1
+   outputs:
+     image_branch_tag: "${{ steps.setup_dependencies.outputs.image_branch_tag }}"
+     operator_specific_tag_for_test: "${{ steps.setup_dependencies.outputs.operator_specific_tag_for_test }}"
+
+  Unit_testing:
+    runs-on: ubuntu-latest
+    name: "Unit testing"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Run unit tests
+        run: |
+          mkdir -p build/_output/reports && chmod 777 build/_output/reports
+          ./build/ci/run_unittests.sh
+
+  Run_OLM_validation:
+    runs-on: ubuntu-latest
+    name: "Run OLM validation"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Run OLM validation
+        run: |
+          ./build/ci/run_olm_validation.sh
+          
+  CSI_Build_and_push_images:
+    name: "CSI Build and push images"
+    runs-on: ubuntu-latest
+    needs:
+      - Unit_testing
+      - Run_OLM_validation
+      - Prepare_env
+    strategy:
+      matrix:
+        image_type: ['nodeagent', 'operator']
```

Comment:
do we want to keep `nodeagent`?

Reply:
I didn't want to break the current procedure

Reply:
I think it's time